vm/src/stdlib/sys.rs

```diff
@@ -590,7 +590,12 @@ mod sys {
     fn unraisablehook(unraisable: UnraisableHookArgs, vm: &VirtualMachine) {
         if let Err(e) = _unraisablehook(unraisable, vm) {
             let stderr = super::PyStderr(vm);
-            writeln!(stderr, "{}", e.as_object().repr(vm).unwrap().as_str());
+            let default = vm.ctx.intern_str("").to_owned();
+            writeln!(
+                stderr,
+                "{}",
+                e.as_object().repr(vm).unwrap_or(default).as_str()
```

Comment:
it might make sense to try and find out what CPython uses here, it might very well be `""` if a repr can't be found.

Reply:
I didn't check cpython side, but if it hangs on CPython, would those kind of thing exist?

Reply:
I was thinking more of a general "what if repr somehow can't be found" case but dunno if that ever happens. I guess `""` would be a sensible default here.

Reply:
I also guess this is just fine for unraisable context - or maybe something repr-error notification.